There are some more references to `version` that need to be changed

```
scripts/ci/pre_commit/pre_commit_update_versions.py:31: error: Module "setup"
has no attribute "version"; maybe "git_version"?  [attr-defined]
    from setup import version  # isort:skip
    ^
scripts/in_container/run_migration_reference.py:34: error: Module "setup" has
no attribute "version"; maybe "git_version"?  [attr-defined]
    from setup import version as _airflow_version
```

--- a/airflow/__init__.py
+++ b/airflow/__init__.py
@@ -52,7 +52,7 @@ from airflow import settings
 
 __all__ = ["__version__", "login", "DAG", "PY36", "PY37", "PY38", "PY39", "PY310", "XComArg"]
 
-__version__ = "2.6.0dev0"
+__version__ = "2.6.0.dev0"
 
 # Make `airflow` an namespace package, supporting installing
 # airflow.providers.* in different locations (i.e. one in site, and one in user

--- a/scripts/ci/pre_commit/common_precommit_utils.py
+++ b/scripts/ci/pre_commit/common_precommit_utils.py
@@ -16,6 +16,7 @@
 # under the License.
 from __future__ import annotations
 
+import ast
 import hashlib
 import os
 import re
@@ -23,6 +24,18 @@ from pathlib import Path
 
 AIRFLOW_SOURCES_ROOT_PATH = Path(__file__).parents[3].resolve()
 AIRFLOW_BREEZE_SOURCES_PATH = AIRFLOW_SOURCES_ROOT_PATH / "dev" / "breeze"
+
+
+def read_airflow_version() -> str:
+    ast_obj = ast.parse((AIRFLOW_SOURCES_ROOT_PATH / "airflow" / "__init__.py").read_text())
+    for node in ast_obj.body:
+        if not isinstance(node, ast.Assign):
+            continue
+
+        if node.targets[0].id == "__version__":  # type: ignore[attr-defined]
+            return ast.literal_eval(node.value)
+
+    raise RuntimeError("Couldn't find __version__ in AST")
 
 
 def filter_out_providers_on_non_main_branch(files: list[str]) -> list[str]:

--- a/scripts/ci/pre_commit/pre_commit_update_versions.py
+++ b/scripts/ci/pre_commit/pre_commit_update_versions.py
@@ -17,23 +17,18 @@
 # under the License.
 from __future__ import annotations
 
-import glob
-import os
 import re
 import sys
-from os.path import abspath, dirname, join
+from pathlib import Path
 
-AIRFLOW_SOURCES_DIR = abspath(join(dirname(__file__), os.pardir, os.pardir, os.pardir))
+sys.path.insert(0, str(Path(__file__).parent.resolve()))  # make sure common_precommit_utils is importable
 
-sys.path.insert(0, AIRFLOW_SOURCES_DIR)
-# flake8: noqa: F401
-
-from setup import version  # isort:skip
+from common_precommit_utils import AIRFLOW_SOURCES_ROOT_PATH, read_airflow_version  # noqa: E402
 
 
-def update_version(pattern: re.Pattern, v: str, file_path: str):
+def update_version(pattern: re.Pattern, v: str, file_path: Path):
     print(f"Checking {pattern} in {file_path}")
-    with open(file_path, "r+") as f:
+    with file_path.open("r+") as f:
         file_content = f.read()
         if not pattern.search(file_content):
             raise Exception(f"Pattern {pattern!r} doesn't found in {file_path!r} file")
@@ -53,13 +48,14 @@ REPLACEMENTS = {
     r"(\(Assuming Airflow version `).*(`\))": "docs/docker-stack/README.md",
 }
 
-print(f"Current version: {version}")
 
 if __name__ == "__main__":
+    version = read_airflow_version()
+    print(f"Current version: {version}")
     for regexp, p in REPLACEMENTS.items():
         text_pattern = re.compile(regexp, flags=re.MULTILINE)
-        files = glob.glob(join(AIRFLOW_SOURCES_DIR, p), recursive=True)
+        files = list(AIRFLOW_SOURCES_ROOT_PATH.glob(p))
         if not files:
             print(f"ERROR! No files matched on {p}")
-        for file in glob.glob(join(AIRFLOW_SOURCES_DIR, p), recursive=True):
+        for file in files:
             update_version(text_pattern, version, file)

--- a/scripts/ci/pre_commit/pre_commit_version_heads_map.py
+++ b/scripts/ci/pre_commit/pre_commit_version_heads_map.py
@@ -28,7 +28,9 @@ PROJECT_SOURCE_ROOT_DIR = Path(__file__).resolve().parent.parent.parent.parent
 
 DB_FILE = PROJECT_SOURCE_ROOT_DIR / "airflow" / "utils" / "db.py"
 
-AIRFLOW_INIT_FILE = PROJECT_SOURCE_ROOT_DIR / "airflow" / "__init__.py"
+sys.path.insert(0, str(Path(__file__).parent.resolve()))  # make sure common_precommit_utils is importable
+
+from common_precommit_utils import read_airflow_version  # noqa: E402
 
 
 def read_revision_heads_map():
@@ -45,21 +47,8 @@ def read_revision_heads_map():
     return revision_heads_map.keys()
 
 
-def read_current_airflow_version():
-
-    ast_obj = ast.parse(open(AIRFLOW_INIT_FILE).read())
-    for node in ast_obj.body:
-        if not isinstance(node, ast.Assign):
-            continue
-
-        if node.targets[0].id == "__version__":
-            return Version(ast.literal_eval(node.value))
-
-    raise RuntimeError("Couldn't find __version__ in AST")
-
-
 if __name__ == "__main__":
-    airflow_version = read_current_airflow_version()
+    airflow_version = Version(read_airflow_version())
     if airflow_version.is_devrelease or "b" in (airflow_version.pre or ()):
         exit(0)
     versions = read_revision_heads_map()

--- a/scripts/in_container/run_migration_reference.py
+++ b/scripts/in_container/run_migration_reference.py
@@ -30,13 +30,13 @@ from typing import TYPE_CHECKING, Iterable
 from alembic.script import ScriptDirectory
 from tabulate import tabulate
 
+from airflow import __version__ as airflow_version
 from airflow.utils.db import _get_alembic_config
-from setup import version as _airflow_version
 
 if TYPE_CHECKING:
     from alembic.script import Script
 
-airflow_version = re.match(r"(\d+\.\d+\.\d+).*", _airflow_version).group(1)  # type: ignore
+airflow_version = re.match(r"(\d+\.\d+\.\d+).*", airflow_version).group(1)  # type: ignore
 project_root = Path(__file__).parents[2].resolve()
 
 

